day_6/questions.md

```diff
@@ -2,12 +2,41 @@
 
 1. In your own words, what is a Class?
 
+A class is a general category of something. Like 'seafood' is a class and 'salmon' is an instance of that class.
+
 1. In relation to a Class, what is a property?
 
+A property is a thing that all members of the class have. Like a property of the class seafood might be 'livesInTheOcean'. I'm hungry.
+
 1. In relation to a Class, what is a method?
 
+A method is a thing that all members of a class can do/have done to it. A method of the seafood class might be 'canBeEaten'.
+
 1. In the space below, create a Car class with at least 2 attributes and 2 methods
 
+class Car {
+  constructor(horsepower, numberOfDoors) {
+    this.horsepower = horsepower;
```

Comment:
How might you rewrite this class so that your methods are utilizing attributes? Remember, we always want to use or modify an attribute within our methods.